test/migrators/calc_interpolation/calc_remove_interpolation.hrx

```diff
@@ -9,14 +9,14 @@ $d: 5;
 // More than one interpolations
 .a { 
     .b: calc($b - #{$c + 1} + #{$d}); 
-    .c: calc(100% - #{$table_title_height + 2px});
+    .c: calc(100% - #{$TABLE_TITLE + 2px});
 }
 
 // Nested
 .a { .b: calc(3 + max(#{$c, 2})); }
 
 // Nested and more interpolations
-.a { .b: calc(#{$b} + max(#{$c, 2})); }
+.a { .b: calc(#{$a-b} + max(#{$c, 2})); }
```

Comment:
Add a declaration for `$a-b`. The migrator _can_ run on files that parse but don't compile, but it's best if our test cases are all valid Sass code the parses and compiles.